tests/test_utils/test_tensor_move.py

```diff
@@ -1,34 +1,54 @@
 import pytest
 
 from colossalai.utils.cuda import get_current_device
-from colossalai.utils.memory_utils.utils import colo_model_data_tensor_move, colo_model_data_tensor_move_inline
+from colossalai.utils.memory_utils.utils import colo_tensor_mem_usage, colo_model_data_tensor_move, colo_model_data_tensor_move_inline, colo_set_process_memory_fraction, colo_cuda_memory_capacity
```

Comment:
You did not test this colo_model_tensor_clone()

Reply:
have added it